doc/latex.rst

```diff
@@ -1148,6 +1140,11 @@ Additional  CSS-like ``'sphinxsetup'`` keys
    set the foreground and background colors for the title as well as the
    LaTeX code for the icon.
 
+.. versionadded:: 7.4.0  Customizability of the :rst:dir:`seealso` and
+   :rst:dir:`todo` directives.
+
+.. versionadded:: 8.1.0 Separate customizability and new defaults for the
+   :dudir:`topic`, contents_, and :dudir:`sidebar` directives.
```

Comment:
```suggestion
.. versionadded:: 7.4.0
   Customizability of the :rst:dir:`seealso` and
   :rst:dir:`todo` directives.

.. versionadded:: 8.1.0
   Separate customizability and new defaults for the
   :dudir:`topic`, contents_, and :dudir:`sidebar` directives.
```

I know we can put the text just after the version, but I find it easier to read/review in the RST if we separate them both.